executor/sort.go

```diff
@@ -131,6 +142,100 @@ func (e *SortExec) Next() (Row, error) {
 	return row, nil
 }
 
+// NextChunk implements the Executor NextChunk interface.
+func (e *SortExec) NextChunk(chk *chunk.Chunk) error {
+	chk.Reset()
+	if !e.fetched {
+		err := e.fetchRowChunks()
+		if err != nil {
+			return errors.Trace(err)
+		}
+		err = e.buildKeyChunks()
+		if err != nil {
+			return errors.Trace(err)
+		}
+		sort.Slice(e.rowPointers, e.newLess)
```

Comment:
https://github.com/golang/go/issues/22504
Seems sometimes `sort.Slice` is slower than `sort.Sort`, can we do some experiment on `sort.Sort` vs. `sort.Slice` ?

Reply:
I did a micro benchmark with the `rowPointer` type slice, and the result show `sort.Slice` is faster.